torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2741,15 +2774,12 @@ def _make_tensor_helper(shape, low=None, high=None):
     OpInfo('index_select',
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            test_inplace_grad=False,
-           skips=(
-               # https://github.com/pytorch/pytorch/issues/49707
-               SkipInfo('TestCommon', 'test_variant_consistency_eager',
-                        dtypes=[torch.float16, torch.bfloat16]),
-               SkipInfo('TestCommon', 'test_variant_consistency_jit', dtypes=[torch.float16, torch.bfloat16]),
-               # index_select does not correctly warn when resizing out= inputs
-               SkipInfo('TestCommon', 'test_out')
-           ),
+           supports_out=False,  # index_select does not correctly warn when resizing out=
```

Comment:
could you file an issue for this and link it in a comment above?

Reply:
https://github.com/pytorch/pytorch/issues/54061